config/tcs_config.toml

```diff
@@ -32,6 +32,7 @@ sleep_interval = "10"
 [Listener]
 listener_port = 1947
 max_work_order_count = 10
+lookup_page_size = 20
```

Comment:
Do we need this constant here also? It is already present in tcf_connector.toml.

Reply:
Yes @Ram-srini, we would need it here as well. The listener handlers make use of this config file.